<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="9" hostname="DESKTOP-4T93PS0" name="automation_testscripts.Doc_Repository" tests="13" failures="0" timestamp="27 May 2019 17:23:20 GMT" time="10.014" errors="4">
  <testcase name="VerifyValidationMessageAfterClickingOnCreateDealButtonWhenHomeRootRopoIsNotSelected" time="5.004" classname="automation_testscripts.Doc_Repository">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.util.Objects.requireNonNull(Unknown Source)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at library.Doc_Repository_page.before_selecting_folder_create_deal(Doc_Repository_page.java:115)
at automation_testscripts.Doc_Repository.VerifyValidationMessageAfterClickingOnCreateDealButtonWhenHomeRootRopoIsNotSelected(Doc_Repository.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- VerifyValidationMessageAfterClickingOnCreateDealButtonWhenHomeRootRopoIsNotSelected -->
  <testcase name="VerifyValidationMessageWhileCreateFolderWhenRootHomeRepoIsNotSelected" time="0.005" classname="automation_testscripts.Doc_Repository">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.util.Objects.requireNonNull(Unknown Source)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at library.Doc_Repository_page.add_folder(Doc_Repository_page.java:62)
at automation_testscripts.Doc_Repository.VerifyValidationMessageWhileCreateFolderWhenRootHomeRepoIsNotSelected(Doc_Repository.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- VerifyValidationMessageWhileCreateFolderWhenRootHomeRepoIsNotSelected -->
  <testcase name="SelectHomeRoot" time="5.004" classname="automation_testscripts.Doc_Repository">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.util.Objects.requireNonNull(Unknown Source)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:106)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
at library.Doc_Repository_page.root(Doc_Repository_page.java:50)
at automation_testscripts.Doc_Repository.SelectHomeRoot(Doc_Repository.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- SelectHomeRoot -->
  <testcase name="VerifyValidationMessageAfterClickingOnCreateDealButtonWhenRootHomeRopoIsSelected" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- VerifyValidationMessageAfterClickingOnCreateDealButtonWhenRootHomeRopoIsSelected -->
  <testcase name="VerifyValidationMessageWhileUploadingDocumentInHomeRootRepo" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- VerifyValidationMessageWhileUploadingDocumentInHomeRootRepo -->
  <testcase name="ClickAddFolderButton" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- ClickAddFolderButton -->
  <testcase name="SendingAFolderNameIsEmpty" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- SendingAFolderNameIsEmpty -->
  <testcase name="VerifySuccessfullMessageAfterCreatingFolder" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- VerifySuccessfullMessageAfterCreatingFolder -->
  <testcase name="SelectCreateFolder" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- SelectCreateFolder -->
  <testcase name="VerifyValidationMessageAfterAfterClickingOnCreateDealButtonWhenCreatedFolderIsNotContainDocuments" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- VerifyValidationMessageAfterAfterClickingOnCreateDealButtonWhenCreatedFolderIsNotContainDocuments -->
  <testcase name="UploadDocuments" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- UploadDocuments -->
  <testcase name="CreateDeal" time="0.000" classname="automation_testscripts.Doc_Repository">
    <skipped/>
  </testcase> <!-- CreateDeal -->
  <testcase name="CloseTab" time="0.001" classname="automation_testscripts.Doc_Repository">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at automation_testscripts.Doc_Repository.CloseTab(Doc_Repository.java:178)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CloseTab -->
</testsuite> <!-- automation_testscripts.Doc_Repository -->
